packages/react-components/src/theme-provider/src/ThemeProvider.tsx

```diff
@@ -38,11 +38,11 @@ export function ThemeProvider({
     as: TriggerType = "div",
     ...rest
 }: ThemeProviderProps) {
-    const [remoteColorScheme, setRemoteColorScheme] = useState();
+    const [remoteColorScheme, setRemoteColorScheme] = useState<ColorSchemeOrSystem>();
```

Comment:
Should be `ColorScheme` instead of `ColorSchemeOrSystem`.

Basically this is intended to work with some kind a dark mode toggle, doesn't make sense to accept "system" here.

Reply:
Wouldn't it make sense that the consumer app lets the user choose to "use system settings". Or should i put system as the default value and overwrite when the user changes the toggle the first time? and never go back to system. In our app at the moment, we have 3 buttons, dark/light/system. I can't enforce the system value at the moment
```
<ThemeProvider colorScheme="system" defaultColorScheme="light" theme="apricot">
```

Reply:
I dont really see any use cases where the user would go back to "system". Will be glad to offer it if we have the use case.

Reply:
Perfect, ill revert those changes